tac/project-updates/2024/2024-Q4-Hyperledger-Identus.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Identus
+parent: 2024
+grand_parent: Project Updates
+---
+Created by Javier Ribó with input from other maintainers
+
+
+# Project Health
+
+The project is still new to the Hyperledger Ecosystem and continues to have steady growth of contributors.
+The project can now be found on the [LFX dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=identus&routedFrom=Github&bestPractice=false&repository=all&dateFilters=Last%20Quarter&dateRange=2024-07-01%20to%202024-09-30&compare=PP&granularity=week&hideBots=true) which can now be checked for health insights. In the last quarter we have seen an increase on the number of github issues created while the active contributors have gone down 8%.
+
+
+# Questions/Issues for the TOC
+
+N/A
+
+# Releases
+
+  * Identus V2.13
+    * Cloud Agent v1.39.0
+    * PRISM Node v2.5.0
+    * Mediator v1.0.0
+    * Apollo Crypto Library v1.4.3
+    * Edge Agent SDK Swift v6.1.0
+    * Edge Agent SDK TypeScript v6.1.0
+    * Edge Agent SDK KMP v4.0.0
+    * Documentation Portal v1.110
+
+  * In the coming days the following releases are planned:
+    * Prism Node 3.0.0
+    * Mediator 1.0.0
```

Comment:
is this a typo since 1.0.0 is already available as part of the v2.13 release?

Reply:
You right the Mediator version v0.15.0 is the one releases with `Identus V2.13`.
The Mediator version v1.0.0 was release after but is backwards compatible.